SQL/0000-00-00-schema.sql

```diff
@@ -724,6 +732,38 @@ INSERT INTO mri_protocol (Center_name,Scan_type,TR_range,TE_range,time_range) VA
   ('ZZZZ',44,'2000-2500','2-5',NULL),
   ('ZZZZ',45,'3000-9000','100-550',NULL);
 
+CREATE TABLE `mri_protocol_group` (
+    `MriProtocolGroupID`   INT(2) UNSIGNED NOT NULL AUTO_INCREMENT,
+    `Name`                 VARCHAR(255)    NOT NULL,
+    PRIMARY KEY (`MriProtocolGroupID`)
```

Comment:
The primary key is the Name.
Unless duplicate name are required.

Reply:
PK using varchar(255) is not the best idea.
If Name must be unique, just use `UNIQUE`

Reply:
I'll leave `MriProtocolGroupID` as the PK and use `UNIQUE` for `Name`.

Typically, this table should have <10 entries. Why not leave the ID to be an `INT(2)`?

Reply:
@nicolasbrossard INT(2) and INT(4) take the same space. They are INT. The parenthesis are for display purpose : 

>  MySQL supports an extension for optionally specifying the display width of integer data types in parentheses following the base keyword for the type. For example, INT(4) specifies an INT with a display width of four digits
https://dev.mysql.com/doc/refman/5.5/en/numeric-type-attributes.html

Reply:
Regarding the varchar(255) primary, I don't think it really matter in that case but I won't argue. My point was that there should be a Primary key on the table and Name should probably be unique.

Reply:
I don't think < 10 protocols is a safe assumption, even if it is the case today.